src/modules/components/asset/assetDataListItem/assetDataListItemStructure/index.ts

```diff
@@ -0,0 +1,7 @@
+import { AssetDataListItemStructure } from './assetDataListItemStructure';
+
+export const AssetDataListItem = {
+    Structure: AssetDataListItemStructure,
+};
+
+export type { IAssetDataListItemStructureProps } from './assetDataListItemStructure';
```

Comment:
The `AssetDataListItem` constant should be defined on the `/assetDataListItem/index.ts` file, like it is done for the `ProposalDataListItem` component. The exports on the `DaoDataListItem` and `MemberDataListItem` are also wrong, could you update them as well on this PR? 🙏🏻